### ISSUE TITLE
cordon task does not exist (anymore)

### ISSUE THREAD

**Issue by JeanPluzo** [blindsidenetworks/scalelite#760]
**Describe the bug**
According to the docs (https://github.com/blindsidenetworks/scalelite#cordon-a-server), there should be a `rake servers:cordon` task. However the `cordon` tasks seems to not be there at all (after doing `rake --tasks`)

**To Reproduce**
Issue the command to cordon a server from Scalelite

***Deployment:***
1. Deployment: docker-compose
2. Versions: Scalelite 1.3.2.1, Ubuntu 20.04.3, Docker 20.10.12, Docker-compose 1.29.2
3. Tools used for reproducing the issue: rake inside container, cli

***Steps to reproduce the behavior:***
1. `docker exec -i scalelite-api bundle exec rake servers:cordon[server-id]`
2. Error: `Don't know how to build task 'servers:cordon' (See the list of available tasks with 'rake --tasks')`

**Expected behavior**
A server should be cordoned (wait until everything in that server has finished and the disable it)

**Additional context**
Is the task missing?
Has it been eliminated from current releases? If so, please update the documentation.
This seems to me like a very useful rake-task for Scalelite. It would be a shame if it was eliminated.
Thanks in advance!

**Comment by jfederico** [comment]
What do you mean with that does not exist anymore?

https://github.com/blindsidenetworks/scalelite/blob/master/lib/tasks/servers.rake#L80


![Screenshot from 2022-03-02 10-39-46](https://user-images.githubusercontent.com/578359/156395398-d350256a-1540-40ca-a1ef-020aa4ad07da.png)

**Comment by JeanPluzo** [comment]
You're right, it IS right there.
Could it be that I've pulled an older version where `cordon` is not yet implemented?
I'm looking at the `docker-compose.yml` file and it says:
`scalelite-api:
    image: ${SCALELITE_DOCKER_IMAGE:-blindsidenetwks/scalelite:v1.1}`
Is the cordon command/function not implemented in version 1.1?
I also looked at `rake --tasks` and the cordon is nowhere to be found in the version I have installed.
Gracias!
